### DISCUSSION TITLE
Add refinement to AnnulusInternalBoundary mesh

### DISCUSSION THREAD

**Discussion by gthyagi** [underworldcode/underworld3-archive#164] [General]
Hi @lmoresi,

Should refinement be added into the AnnulusInternalBoundary mesh, and what is the reason behind the absence of a refinement parameter in the current AnnulusInternalBoundary mesh?

**Comment by lmoresi** [comment]
should be added - it's probably just left over from when we added the refinement parameter to the mesh as a whole. You will need to make sure that the refinement callback function adjusts the internal points to lie at the correct radius.
